### PR TITLE
Replacing the class references to the webmozarts/path-util

### PR DESCRIPTION
**Pull request by mamazu**
Since we already remove the package dependency to the webmozarts/path-util package. We should also remove the references in code.

This isn't a big issue since the package is still in the pslam dependency. So it's still installed somehow.

### CODE REVIEW

File: lib/Indexer/Extension/IndexerExtension.php [hunk 1]
```diff
@@ -119,17 +117,6 @@ public function load(ContainerBuilder $container): void
...
-    {
-        $builder = ReflectorBuilder::create();
-        foreach (array_keys($container->getServiceIdsForTag(WorseReflectionExtension::TAG_SOURCE_LOCATOR)) as $serviceId) {
-            $builder->addLocator($container->get($serviceId), 128);
-        }
-        $builder->enableCache();
-
-        return $builder->build();
-    }
-
```
Comment (mamazu) [inline lib/Indexer/Extension/IndexerExtension.php:132]: Phpstan has complained that this methods has no use so I removed it. And it seems like it's fine.

File: lib/Extension/ClassMover/Application/ClassCopy.php [hunk 2]
```diff
@@ -72,7 +72,7 @@ public function copyFile(ClassCopyLogger $logger, string $srcPath, string $destP
             // if the src is not the same as the globbed src, then it is a wildcard
             // and we want to append the filename to the destination
             if ($srcPath !== $globPath) {
-                $globDest = Path::join($destPath, Path::getFilename($globPath));
+                $globDest = Path::join($destPath, basename($globPath));
```
Comment (dantleech) [inline lib/Extension/ClassMover/Application/ClassCopy.php:75]: hm, which this change?

Comment (dantleech) [reply lib/Extension/ClassMover/Application/ClassCopy.php:75]: ok, old version did basically the same thing.

**Review by dantleech — APPROVED** [review]

**Comment by dantleech** [comment]
great, thanks!

File: lib/Extension/ClassMover/Application/ClassCopy.php (hunk 2 above)
Comment (mamazu) [reply lib/Extension/ClassMover/Application/ClassCopy.php:75]: Yeah, I basically replaced the function by it's implementation.

**Pull request merged**